client/.storybook/config.js

```diff
@@ -0,0 +1,4 @@
+import { configure, addDecorator } from '@storybook/react';
```

Comment:
DeepScan fails because of this
```suggestion
import { addDecorator } from '@storybook/react';
```